src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    double getArea();
+
```

Comment:
remove the last empty line in all classes/interfaces/methods where it is present